pure.zsh

```diff
@@ -227,9 +227,9 @@ prompt_pure_async_callback() {
 			prompt_pure_git_dirty=$output
 			prompt_pure_preprompt_render
 
-			# when prompt_pure_git_delay_dirty_check is set, the git info is displayed in a different color, this is why the
-			# prompt is rendered before the variable is (potentially) set
-			(( $exec_time > 2 )) && prompt_pure_git_delay_dirty_check=$EPOCHSECONDS
+			# when prompt_pure_git_last_dirty_check_timestamp is set, the git info is displayed in a different color,
+			# this is why the prompt is rendered before the variable is (potentially) set
```

Comment:
Hmm, your editing this comment made me realize that it itself is very confusing. I think it would be best to remove the whole `this is why the prompt is rendered before the variable is (potentially) set` since prompt rendering is not a result of this variable being set or not.